sqlite/ext/comdb2/tunables.c

```diff
@@ -84,21 +88,29 @@ static int systblTunablesDisconnect(sqlite3_vtab *pVtab)
     return SQLITE_OK;
 }
 
+static void jumpToNextVisibleTunable(systbl_tunables_cursor *cur)
```

Comment:
This works if the last tunable is hidden or we access past the end?

Reply:
That's checked under systblTunablesNext(). While verifying this, I found another issue though.